data/slash/slash.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package proto;
+
+option go_package = "slash";
+option (gogoproto.stable_marshaler_all) = true;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "github.com/ElrondNetwork/elrond-go-core/data/block/blockV2.proto";
+
+enum SlashingType {
+  None = 0;
+  MultipleProposal = 1;
+  MultipleSigning = 2;
+}
+
+enum ThreatLevel {
+  Low = 0;
+  Medium = 1;
+  High = 2;
+}
+
+message Headers{
+  repeated HeaderV2 Headers = 1;
```

Comment:
can you make this nullable false so that it generates the field as a  []*HeaderV2 instead of *[]*HeaderV2

Reply:
Yes, removed pointer to slice for both proofs